scripts/run_examples_dev

```diff
@@ -4,6 +4,8 @@ readonly SCRIPTS_DIR="$(dirname "$(readlink -f "$0")")"
 # shellcheck source=scripts/common
 source "${SCRIPTS_DIR}/common"
 
+export TSAN_OPTIONS="halt_on_error=1 report_atomic_races=0"
```

Comment:
Maybe add a link to https://github.com/google/sanitizers/wiki/ThreadSanitizerFlags or some other reference, in a comment.

Reply:
Good idea, done.